hasktorch/src/Torch/Functional.hs

```diff
@@ -551,9 +551,14 @@ maxPool2d
     -> Bool -- ^ ceil mode
     -> Tensor -- ^ input
     -> Tensor -- ^ output
-maxPool2d kernelSize stride padding dilation ceilMode self =
+maxPool2d (kernelSize0, kernelSize1) (stride0, stride1) (padding0, padding1) (dilation0, dilation1) ceilMode self =
```

Comment:
Wouldn't it be nicer to have a helper like `asList :: (Int, Int) -> [Int]` and simply apply that to every argument without explicit unpacking?

Reply:
Yeah, sure will get this done.